code/modules/research/designs.dm

```diff
@@ -45,3 +45,5 @@ other types of metals and chemistry for reagents).
 	var/list/reagents_list = list()			//List of reagents. Format: "id" = amount.
 	var/maxstack = 1
 	var/lathe_time_factor = 1			//How many times faster than normal is this to build on the protolathe
+	var/list/consolelimit = list()		//Remove design from consoles with listed ID
```

Comment:
Call this `console_blacklist`. The term `consolelimit` makes it sound like a whitelist.